### PR TITLE
Chevron icons

### PR DESCRIPTION
**Pull request by SabaMallah**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -42,3 +45,20 @@ function hideCartHandler() {
   cart.style.transform = "translateX(100%)";
   cartOverlay.style.visibility = "hidden";
 }
+
+for (let i = 0; i < allChevronUpBtns.length; i++)
```
Comment (imambux) [inline app.js]: you can also shorten the code by writing like this:
allChevronUpBtns.forEach((btn) => { btn.addEventListener...})

Also, the reason to use forEach over index based for loop is that you actually don't need index so why event use it?

Also the same for all down btns.

File: app.js [hunk 2]
```diff
@@ -42,3 +45,20 @@ function hideCartHandler() {
...
+function allChevronUpBtnsHandler(event) {
+  event.target.nextElementSibling.innerHTML =
+    parseInt(event.target.nextElementSibling.innerHTML) + 1;
+}
+
+for (let i = 0; i < allChevronDownBtns.length; i++)
+  allChevronDownBtns[i].addEventListener("click", allChevronDownBtnsHandler);
+
+function allChevronDownBtnsHandler(event) {
+  if (event.target.previousElementSibling.innerHTML > 0)
```
Comment (imambux) [inline app.js]: Now that I think about it, 0 amount for item doesn't make sense. I mean 0 item actually means that item is removed. So, let's make it 1 as the smallest possible amount.

**Review by imambux — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (SabaMallah) [reply app.js]: I have fixed it.

File: app.js (hunk 2 above)
Comment (SabaMallah) [reply app.js]: Thanks , I did not notice it but i have set it to 1 now.

**Review by imambux — APPROVED** [review]
Nice 👌🏽

**Pull request merged**